pkg/crc/config/settings.go

```diff
@@ -107,6 +109,10 @@ func RegisterSettings(cfg *Config) {
 
 	cfg.AddSetting(KubeAdminPassword, "", ValidateString, SuccessfullyApplied,
 		"User defined kubeadmin password")
+	cfg.AddSetting(IngressHTTPPort, 80, ValidatePort, RequiresHTTPPortChangeWarning,
+		"Consent to use alternative http port in case 80 is in use (1024-65535, default: 80)")
```

Comment:
`Consent to use`? "Alternative HTTP port to use for OpenShift ingress/routes in case port 80 is already in use on the host" ? But this could be too long.

Reply:
Yeah I can put it, too long shouldn't be issue if it make more clear message to user.